_drafts/2023-02-13-cheating-the-system-for-fun-and-profit.md

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+permalink: cheating-the-system-for-fun-and-profit
+title: 'Cheating the System for Fun and Profit'
```

Comment:
This title is not very descriptive, consider adding a subtitle?

Reply:
Haha good point, I held off on this but really should. Thanks for the nudge!

Reply:
Always be thinking of that SEO!

Reply:
Nahhh 😛